### PR TITLE
DolphinQt2: don't use custom delegate for GameList

### PR DESCRIPTION
**Pull request by ligfx**
The default delegate correctly handles text, padding, text eliding, and pixmaps of any devicePixelRatio.

Before:

screen shot 2017-05-26 at 5 10 36 pm

After:

screen shot 2017-05-26 at 5 24 48 pm

### CODE REVIEW

File: Source/Core/DolphinQt2/GameList/GameListModel.cpp [hunk 1]
```diff
@@ -5,6 +5,8 @@
 #include "DolphinQt2/GameList/GameListModel.h"
 #include "DolphinQt2/Resources.h"
 
+static const QSize GAMECUBE_BANNER_SIZE(96, 32);
```
Comment (lioncash) [inline Source/Core/DolphinQt2/GameList/GameListModel.cpp]: `static` isn't necessary with `const`.

Comment (ligfx) [inline Source/Core/DolphinQt2/GameList/GameListModel.cpp]: Ah cool, didn't know that!

Comment (lioncash) [inline Source/Core/DolphinQt2/GameList/GameListModel.cpp]: If you want the full listing of divergences between C and C++, check out appendix C in the C++ standard. It also lists the rationales behind them as well =)

Comment (ligfx) [inline Source/Core/DolphinQt2/GameList/GameListModel.cpp]: Appendix "C," I hope that was on purpose.

**Comment by MayImilae** [comment]
LGTM!

**Pull request merged**